rest-server/src/components/mapbox/mapboxControllers.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { fetchSingleUserHelper } from '../users/userHelpers';
+
+const mapboxAPIKey = process.env.mapboxAPIKey || 'MUST-REPLACE-WITH-APIKEY-TO-PASS-TEST';
+
+const fetchUserLocationCoordinates = async (req, res) => {
+  try {
+    const singleUser = await fetchSingleUserHelper(req.params);
+    const loc = singleUser.address.split(' ').join('%20');
```

Comment:
When constructing URLs you can encode the string entirely if you need to (https://www.w3schools.com/jsref/jsref_encodeURI.asp).

Reply:
Great suggestion, I fiddled with the string to come up with my manual method. I have replaced it with JavaScript's global encodeURI function.